pages/signup/style.css

```diff
@@ -30,3 +30,49 @@
 #signup-main .btn-primary {
   justify-content: center;
 }
+
+.content-image{
```

Comment:
could you add the main id #signup-main to add more specificity to this stylesheet